### DISCUSSION TITLE
How do I force a row to replace all the column header names?

### DISCUSSION THREAD

**Discussion by blais** [saulpw/visidata#856] [Q&A] [answered]
I have opened a table that contains multiple headers. I'm selection a subset of rows whose header doesn't match the top header. Is there a way to select a row and make that the names of all the columns, i.e. replace the header?  g^ doesn't work
Or equivalently, is there a way to reset the header / column names entirely?

**Comment by blais** [comment, marked as answer]
Oh never mind... gz^ works.

**Comment by alecstein** [comment]
`g^` also worked for me.
